src/Resources/ResourceManager/Components/Constants.cs

```diff
@@ -147,7 +152,7 @@ public static class Constants
         /// <summary>
         /// The management group resource type.
         /// </summary>
-        public static readonly string MicrosoftManagementGroupDefinitionType = Constants.MicrosoftManagementNamespace + "/managementgroups";
+        public static readonly string MicrosoftManagementGroupDefinitionType = Constants.ManagementGroupTypeName + "/managementgroups";
```

Comment:
>Definition [](start = 62, length = 10)

nit - I think you should drop "definition" from the name. There's really no concept of "Definition" in MGs

Reply:
did some refactoring and removed this constant.